README.md

```diff
@@ -0,0 +1,344 @@
+# Building a Crypto News Website using the Microsoft Azure App Service and MongoDB Atlas
+
+Who said creating a website has to be hard?
+
+Writing the code, persisting news, hosting the website.
+A decade ago this might have been a lot of work.
+These days, thanks to [Microsoft Blazor](https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor),
+[Microsoft Azure App Service](https://azure.microsoft.com/en-us/products/app-service/#overview)
+and [MongoDB Atlas](https://www.mongodb.com/atlas/database) you can get started in minutes.
+And finish it equally fast!
+
+In this tutorial I will walk you through:
+
+* setting up a new Blazor project,
+* creating a new page with a simple UI,
+* creating data in Atlas,
+* showing those news on the website and
+* making the website available by using Azure App Service to host it.
+
+All you need is this tutorial and the following pre-requisites, but if you prefer to just read along for now,
+check out the [GitHub repository for this tutorial](https://github.com/mongodb-developer/crypto-news-website)
+where you can find the code and the tutorial.
+
+## Pre-requisites for this tutorial
+
+Before we get started, here is a list of everything you need while working through the tutorial.
+I recommend getting everything set up first so that you can seamlessly follow along.
+
+* [Download and install the .NET framework](https://dotnet.microsoft.com/en-us/download).
+  For this tutorial I am using .NET 7.0.102 for Windows but any .NET 6.0 or higher should do.
+* [Download and install Visual Studio](https://visualstudio.microsoft.com/downloads/).
+  I am using the 2022 Community edition, version 17.4.4, but any 2019 or 2022 edition will be ok.
+  Make sure to install the `Azure development` workload as we will be deploying with this later.
+  If you already have an installed version of Visual Studio, go into the Installer and click `modify` to find it.
+* [Sign up for a free Microsoft Azure account](https://azure.microsoft.com/en-us/free/).
+* [Sign up for a free MongoDB Atlas account](https://account.mongodb.com/account/register).
+
+## Creating a new Microsoft Blazor project that will contain our Crypto News Website
+
+Now that the pre-requisites are out of the way, let's start by creating a new project.
+
+![01_get_started.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/01_get_started_88f36b9147.jpg)
```

Comment:
Don't forget to include "alt text" in your images.

Reply:
Actually made a mistake there and added it after the URL. It's fixed with 147423045eadb3e5a9eaab25a07ee414790f6486